### DISCUSSION TITLE
Swagger UI has a title of "404 - Resource Not Found"?

### DISCUSSION THREAD

**Discussion by kenyee** [quarkusio/quarkus#23192] [Q&A] [answered]
Started hooking up openapi docs after adding the "io.quarkus:quarkus-smallrye-openapi" artifact and get this interesting looking screen at localhost:8080/swagger-ui 🤔 
No errors in the quarkusDev console and the network trace doesn't show anything like a bad resource fetch...this endpoint is returning this.  The docs could also use a bit more info on what annotations you can add to fields of data models in the API FWIW...

<img width="1036" alt="Screen Shot 2022-01-25 at 2 57 40 PM" src="https://user-images.githubusercontent.com/586670/151050236-5c1a1c59-4b98-4c8f-9fba-e5ae7bcaf1e2.png">

Has anyone seen this and know what might be causing it?  I didn't see any discussion posts or issues on this..

Can repro w/ the same project as the one I posted in this thread: https://github.com/quarkusio/quarkus/discussions/23181

[a comment by an automated account is not shown]

**Comment by gsmet** [comment, marked as answer]
If you use the default config, the URL is `/q/swagger-ui`.

**Comment by kenyee** [comment]
that works perfectly..thanks
super weird that /swagger-ui partially works as above... 🤔

**Reply by gsmet** [reply]
What do you mean `partially works`? It doesn't work. You just get the nice 404 error page.

**Comment by kenyee** [comment]
the resources are described on the same page...

**Comment by kenyee** [comment]
ohhh...the error page includes the resources....  that's what's confusing :-)

Your error page is too useful, so it doesn't look like a normal error page...a 404 is usually a blank page 😂
